<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
	<script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js"></script>
	<?php include "header.html";?>
	<title>Major Lab - Research</title>
	<script type="text/javascript" src="js/google-analytics.js"></script>
</head>
<body>
	<div id="wrap">
		<?php include "navbar.html";?>
		<div id="stripe2">
	
			<div id="str2">
				<div class="innerdiv">
					<div class="pageHeader">
						<h1>Research</h1>
					</div>
	    
					<div class="content" style="font-size:14px">
						<p class="paragraph"> Although initiated by genetic mutation, the unchecked proliferation, aberrant differentiation, and altered motility of cancer cells 
							depends upon the integrity and activation state of specific signal transduction pathways. Our laboratory is interested in understanding 
							how alterations in these signaling pathways contribute to human cancer, and whether exploitation of that understanding can aid in the 
							development of new diagnostics, prognostics and therapeutic intervention strategies. To this end, we employ a global “systems level” 
							integrative discovery platform, one that has as a foundation mass spectrometry-based proteomic interaction networks. More specifically, 
							through LC/MS/MS, we define the physical interaction network for a signaling pathway of oncogenic interest. By small molecule and functional 
							genomic screening, we then annotate the human genome for functional contribution to the pathway of interest. Integration of these data with 
							cancer-associated mutation data and cancer-associated gene expression data yields a powerful tool for oncogenic discovery—a cancer annotated 
							physical/functional map for a specific signaling pathway of interest. The models and hypotheses produced though integrative screening are 
							challenged through mechanistic studies employing cultured human cancer cells, zebrafish, mice and in vitro biochemical systems. With this 
							general approach, we are currently pursuing the following projects: </p>
	      
								<details>
									<summary class="divider">Mechanistic Studies of Wnt/&beta;-catenin Signaling</summary>
									<p> Of the relatively small number of signaling pathways that function as master regulators of development, adult tissue homeostasis and cancer,
										the &beta;-catenin dependent Wnt pathway (Wnt/&beta;-catenin) figures prominently; it regulates the growth and fate of neoplastic cells in tissues of diverse
										origin, notably the colon, kidney, breast and skin. My group has performed an array of proteomic and functional genomic studies of WNT signaling, 
										including protein-protein interaction screens, kinase enrichment profiling, phospho-proteomics, siRNA and haploid mutagenesis loss-of-function screens, 
										and more recently, novel gain-of-function screens. Integration of these data has and continues to reveal mechanistic insight and new disease-relevant 
										regulators of pathway activity. As an example, our gain-of-function genomic screens demonstrated that the FOXP1 transcription factor activates &beta;-catenin
										dependent transcription. Proteomic analyses demonstrated that FOXP1 binds the &beta;-catenin transcriptional complex on chromatin. Disease-focused studies in 
										mice and human clinical samples demonstrated that FOXP1 overexpression in B-cell lymphoma activates WNT signaling to promote tumor growth. In more recent work, 
										we discovered that the ubiquitin-specific protease (USP6) deubiquitylates the WNT receptor to govern its endocytosis. We also discovered a WNT-driven negative 
										feedback loop that activates the AAK1 kinase to promote endocytosis of LRP6.</p>
								</details>
								<details>
									<summary class="divider">KEAP1 and NRF2 Signal Transduction in Cancer</summary>
									<p> KEAP1 is an E3 ubiquitin ligase important for cellular defense against oxidative stress, and in that context contributes fundamentally to aging, 
										neurodegeneration, and a myriad of human cancers, most notably lung cancer. KEAP1 functions by ubiquitylating the NRF2 transcription factor, 
										resulting in NRF2 degradation. In cancer, mutations within NRF2 or KEAP1 result in constitutive NRF2-driven expression of cytoprotective genes—this occurs 
										in ~30% of lung cancer. In an effort to better understand KEAP1/NRF2, we were the first to define and validate a KEAP1 protein-protein interaction network. 
										This revealed a group of KEAP1-associated proteins that competitively displaces NRF2, driving pathway activation. Our work has also begun to connect KEAP1 
										genotype with phenotype, wherein we functionally and biochemically characterized 19 lung cancer KEAP1 mutations. This revealed that ~half of the KEAP1 cancer 
										mutations are hypomorphic, and surprisingly retain the ability to ubiquitylate NRF2.  Ongoing work is focused on the mechanism by which these ‘ANCHOR’ mutants 
										impair KEAP1-dependent degradation (but not ubiquitylation) of NRF2 function. Based off of proteomic, functional genomic and pharmacological screens, we are 
										also pursuing novel NRF2-independent functions for KEAP1, including a cell cycle phenotype relevant for lung cancer cell proliferation. Most recently, we have 
										created a new NRF2-active mouse model, allowing us to test the importance of our discoveries and the various biological impacts of NRF2 in a physiologically 
										relevant system.</p>
								</details>
	      
								<details>
									<summary class="divider">Computational Proteomics and Functional Genomic Technologies</summary>
									<p> A central tenet of my research program is discovery-based science, founded largely upon mass spectrometry-driven proteomics, functional genomics 
										and disease annotation. The integration of these disparate data types eliminates false positives, rescues false negatives, and highlights neighborhoods 
										within the global network of phenotypic, mechanistic and disease importance. Our ability to integrate and evaluate large datasets has improved dramatically 
										in recent years. My team developed a machine learning approach to probabilistically score and functionally annotate protein-protein interactions—this is the 
										most accurate scoring platform currently available (Spotlite). We also developed an improved MS data acquisition simulator, which facilitates the development 
										of new algorithms for improved protein sequencing. Most recently, we developed novel algorithms to approximate isotopic distributions of MS2 fragment ions; 
										this allows decomplexing of chimeric spectra and improved peptide identification. Within the proteomics arena, we have devoted great effort to kinase enrichment 
										mass spectrometry to capture, identify and quantify the kinome in a single mass spectrometry run. In a recent example, we used this approach in a competitive 
										fashion to define the kinase target landscape for FDA-approved kinase inhibitors. We are also developing new functional genomic screening technologies, but rather
										than continue with the more traditional loss-of-function approaches (siRNA, haploid mutagenesis), our efforts have focused on gain-of-function genome annotation. 
										To this end, we developed an arrayed lentiviral clone library and a mass spectrometry-coupled genome-wide hypermorphic functional annotation technology called 
										CDt/MS (listed under the WNT section). CrisprA and CrisprI screens are also being employed.</p>
								</details>

								<details>
									<summary class="divider">Protein-protein Interaction Networks and E3 Ubiquitin Ligase Substrate Discovery</summary>
									<p> E3 ubiquitin ligase complexes provide specificity and catalysis for the transfer of ubiquitin to target proteins, a post-translational modification that 
										results in proteasome-mediated degradation, altered subcellular localization or changes in protein interaction. Traditional pull-down/MS approaches fail 
										to identify many E3 substrates, in part because the E3 complex is catalytic in action and because substrates are often short-lived. We use pharmacological 
										and genetic approaches to stabilize the E3-substrate interaction. To date, we have identified known and novel substrates for the &beta;TrCP, KEAP1 and RAD18 E3 
										ubiquitin ligases. Work in my laboratory is exploiting these system to identify substrates for uncharacterized E3 ubiquitin ligases, specifically those with 
										established connections to oxidative stress signaling, Wnt signaling and human disease.  Beyond E3 ligases, we are working within a larger team to illuminate 
										the understudied kinome, a set of 162 kinases that remain relatively uncharacterized. My team is defining the protein-protein interaction and proximity networks 
										for these kinases. Additionally, we are ‘binning’ the understudied kinases into signal transduction pathways using arrayed ORF screens and Crispr-based gain and 
										loss-of-function screens.</p>
								</details>
	      
								<details>
									<summary class="divider">Genetic Screening Platforms</summary>
									<p>In addition to this integrative analysis of signaling, we are also actively working to develop a human somatic cell forward-genetic 
										screening platform. Eukaryotic cells sense and interpret extracellular cues through a highly integrated network of intracellular signaling 
										pathways. The identification of these pathways and their constituent proteins can be largely attributed to genetic screens in yeast, 
										Drosophila, c-elegans and zebrafish, each of which permits random mutagenesis screening and clonal isolation in a homozygous state. 
										RNA interference provides analogous loss-of-function technologies in mammalian systems, and by doing so has revolutionized the significance 
										and speed of our discoveries. However, RNA interference-based screens are expensive and fraught with artifact, owing in part to off-target 
										effects and incomplete silencing. An exciting and ongoing effort in my lab is to realize somatic cell genetic screens in human cells. 
										We are pursuing this goal through retroviral insertional mutagenesis of stable near-haploid human cell lines. Leveraging our existing 
										expertise in proteomic and functional genomic analysis of signal transduction, we are employing this haploid screening platform to 
										comprehensively identify proteins and microRNAs required for the following signaling pathways, each of which functions as a master 
										regulator of human disease and human development: TGF&beta;, Wnt/&beta;-catenin, retinoic acid, NF-&kappa;B and hedgehog. A systems-level integration 
										of the resulting functional data with protein-protein interaction networks, genome-wide association data and transcriptional signatures 
										will reveal targets of diagnostic, prognostic and therapeutic value. Considering only the haploid screening approach, success promises 
										rapid, unbiased and inexpensive complete loss-of-function phenotypic screening of human cells, and therefore has the potential to 
										transform the experimental strategies taken in both basic and applied sciences.</p>
								</details>
											<!--<p class="divider-blue small">Clinical Interests</p>
												<div class="ul-div">
												<ul class="topmenu">
												<li> <span>Non-small cell lung cancer</span> </li>
												<li> <span>Wilms' tumor</span> </li>
												<li> <span>Aging</span> </li>
												</ul>
												</div>
												<div style="clear:both"></div>-->
												<p class="divider">Supported by:</p>
												<div class="ul-div no-bullet" style="margin-left:10px; margin-top:10px;">
													<ul class="ul-images">
														<li>
															<a href="http://www.kimmel.org" target="_blank"><img src="img/funding/kimmel.png" alt=""/></a>
														</li>
														<li>
															<a href="http://commonfund.nih.gov/newinnovator" target="_blank"><img src="img/funding/NIH.png" alt=""/></a>
														</li>
														<li>
															<a href="http://www.golfersagainstcancer.org" target="_blank"><img src="img/funding/golfers.png" alt=""/></a>
														</li>
														<li>
															<a href="http://www.cancer.org/" target="_blank"><img src="img/funding/ACS.jpg" alt=""/></a>
														</li>
														<li>
															<a href="http://www.curesarcoma.org/" target="_blank"><img src="img/funding/sarcoma.jpg" alt=""/></a>
														</li> 
													</ul>
												</div>
												<div style="clear:both"></div>
												<div class="ul-div no-bullet" style="margin-left:10px; margin-top:10px;">
													<ul class="ul-images">
														<li>
															<a href="https://druggablegenome.net/" target="_blank"><img src="img/funding/dark_kinome.png" alt=""/></a>
														</li>
													</ul>
												</div>
												<div style="clear:both"></div>
											</div>
										</div>
									</div>
								</div>
							</div>
							<?php include "footer.html";?>
							<script type="text/javascript" src="js/jquery.details.js"></script>
							<script type="text/javascript">
							$(function() {
								$('html').addClass($.fn.details.support ? 'details' : 'no-details');
								$('details').details();
							});
							</script>
						</body>
						</html>
