<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
	<script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js"></script>
	<?php include "header.html";?>
	<title>Major Lab - Research</title>
	<script type="text/javascript" src="js/google-analytics.js"></script>
</head>
<body>
	<div id="wrap">
		<?php include "navbar.html";?>
		<div id="stripe2">
	
			<div id="str2">
				<div class="innerdiv">
					<div class="pageHeader">
						<h1>Research</h1>
					</div>
	    
					<div class="content" style="font-size:14px">
						<p class="paragraph"> Although initiated by genetic mutation, the unchecked proliferation, aberrant differentiation, and altered motility of cancer cells 
							depends upon the integrity and activation state of specific signal transduction pathways. Our laboratory is interested in understanding 
							how alterations in these signaling pathways contribute to human cancer, and whether exploitation of that understanding can aid in the 
							development of new diagnostics, prognostics and therapeutic intervention strategies. To this end, we employ a global “systems level” 
							integrative discovery platform, one that has as a foundation mass spectrometry-based proteomic interaction networks. More specifically, 
							through LC/MS/MS, we define the physical interaction network for a signaling pathway of oncogenic interest. By small molecule and functional 
							genomic screening, we then annotate the human genome for functional contribution to the pathway of interest. Integration of these data with 
							cancer-associated mutation data and cancer-associated gene expression data yields a powerful tool for oncogenic discovery—a cancer annotated 
							physical/functional map for a specific signaling pathway of interest. The models and hypotheses produced though integrative screening are 
							challenged through mechanistic studies employing cultured human cancer cells, zebrafish, mice and in vitro biochemical systems. With this 
							general approach, we are currently pursuing the following projects: </p>
	      
							<details>
								<summary class="divider">Wnt/&beta;-catenin Signal Transduction</summary>
								<p> Of the relatively small number of signaling pathways that function as master regulators of development, 
									adult tissue homeostasis and cancer, the &beta;-catenin dependent Wnt pathway (Wnt/&beta;-catenin) figures prominently; it regulates the growth and fate 
									of neoplastic cells in tissues of diverse origin, notably the colon, kidney, breast and skin. Integrative molecular analysis of Wnt/&beta;-catenin 
									signaling in kidney and colon cancer models yielded numerous discoveries of mechanistic and clinical importance. Work in my lab is now focusing 
									on an integrative analysis of Wnt signaling in pancreatic adenocarcinoma and lung cancer. For both cancer models, independent functional 
									genomic screens have been completed and have identified numerous novel regulators, some of which are mutated in human cancer. In addition,
									we are looking more closely at a family of proteins I discovered to regulate Wnt signaling in Wilms tumor. Specifically, we are working to 
									understand the mechanism(s) by which members of the WTX gene family regulate Wnt signaling and oncogenesis.</p>
								</details>
								<details>
									<summary class="divider">Keap1/Nrf2 Signal Transduction</summary>
									<p>Keap1 is an E3 ubiquitin ligase important for cellular defense against genotoxic stress, and in that context 
										contributes fundamentally to aging and a myriad of human cancers, most notably lung cancer. Keap1 functions by ubiquitinating the Nrf2 
										transcription factor, which ultimately results in the proteosomal degradation of Nrf2. In an effort to better understand Keap1 in an oncogenic 
										contect, I completed quantitative proteomic analysis of the Keap1 protein complex as well as functional genomic screen of the Keap1/Nrf2 
										pathway. The resulting integrative map has identified numerous novel proteins which both physically associate with keap1 as well as functional 
										regulate Keap1/Nrf2 signaling. Ongoing work in the lab is focused on understanding the mechanisms by which these proteins control Keap1 function 
										as well as uncovering new cancer connections.</p>
									</details>
	      
									<details>
										<summary class="divider">E3 Ubiquitin Ligase Substrate Identification</summary>
										<p>E3 ubiquitin ligase complexes provide specificity and catalysis for the transfer of ubiquitin 
											to target proteins, a post-translational modification that results in proteosome-mediated degradation, altered subcellular localization or 
											changes in protein interaction. As such, E3 ubiquitin ligases regulate every facet of cell biology, and importantly, are frequently perturbed 
											in disease states such as cancer. While expanding my protein-protein interaction network for the Wnt/&beta;-catenin pathway, I performed tandem 
											affinity purification and mass spectrometry on &beta;TrCP, the E3 ubiquitin ligase responsible for &beta;-catenin degration. As the E3 complex is 
											catalytic in action, many of the known &beta;TrCP substrates were not identified by LC/MS/MS. As such I designed and implemented a novel approach 
											which stabilizes the E3-substrate interaction. Using this strategy, I have identified novel substrates for both the &beta;TrCp and Keap1 E3 
											ubiquitin ligases. Work in my laboratory is exploiting this system to identify substrates for uncharacterized E3 ubiquitin ligases, 
											specifically those with established connections to human disease. Follow-up experiments require biochemical purification of target 
											substrate following gain-of-function and/or loss-of-function of the E3, in vitro analysis of the ubiquitination reaction, and histological 
											assessment of substrate expression in patient tissue samples. Of great importance, and in fact the greatest challenge, is the demonstration 
											of biological significance for the E3-substrate relationship. To address this, my lab implements an array of technologies encompassing cDNA 
											microarray analysis, proteomics and whole organism studies.</p>
										</details>
	      
										<details>
											<summary class="divider">Genetic Screening Platforms</summary>
											<p>In addition to this integrative analysis of signaling, we are also actively working to develop a human somatic cell forward-genetic 
												screening platform. Eukaryotic cells sense and interpret extracellular cues through a highly integrated network of intracellular signaling 
												pathways. The identification of these pathways and their constituent proteins can be largely attributed to genetic screens in yeast, 
												Drosophila, c-elegans and zebrafish, each of which permits random mutagenesis screening and clonal isolation in a homozygous state. 
												RNA interference provides analogous loss-of-function technologies in mammalian systems, and by doing so has revolutionized the significance 
												and speed of our discoveries. However, RNA interference-based screens are expensive and fraught with artifact, owing in part to off-target 
												effects and incomplete silencing. An exciting and ongoing effort in my lab is to realize somatic cell genetic screens in human cells. 
												We are pursuing this goal through retroviral insertional mutagenesis of stable near-haploid human cell lines. Leveraging our existing 
												expertise in proteomic and functional genomic analysis of signal transduction, we are employing this haploid screening platform to 
												comprehensively identify proteins and microRNAs required for the following signaling pathways, each of which functions as a master 
												regulator of human disease and human development: TGF&beta;, Wnt/&beta;-catenin, retinoic acid, NF-&kappa;B and hedgehog. A systems-level integration 
												of the resulting functional data with protein-protein interaction networks, genome-wide association data and transcriptional signatures 
												will reveal targets of diagnostic, prognostic and therapeutic value. Considering only the haploid screening approach, success promises 
												rapid, unbiased and inexpensive complete loss-of-function phenotypic screening of human cells, and therefore has the potential to 
												transform the experimental strategies taken in both basic and applied sciences.</p>
											</details>
											<!--<p class="divider-blue small">Clinical Interests</p>
												<div class="ul-div">
												<ul class="topmenu">
												<li> <span>Non-small cell lung cancer</span> </li>
												<li> <span>Wilms' tumor</span> </li>
												<li> <span>Aging</span> </li>
												</ul>
												</div>
												<div style="clear:both"></div>-->
												<p class="divider">Supported by:</p>
												<div class="ul-div no-bullet" style="margin-left:10px; margin-top:10px;">
													<ul class="ul-images">
														<li>
															<a href="http://www.kimmel.org" target="_blank"><img src="img/funding/kimmel.png" alt=""/></a>
														</li>
														<li>
															<a href="http://commonfund.nih.gov/newinnovator" target="_blank"><img src="img/funding/NIH.png" alt=""/></a>
														</li>
														<li>
															<a href="http://www.golfersagainstcancer.org" target="_blank"><img src="img/funding/golfers.png" alt=""/></a>
														</li>
														<li>
															<a href="http://www.cancer.org/" target="_blank"><img src="img/funding/ACS.jpg" alt=""/></a>
														</li>
														<li>
															<a href="http://www.curesarcoma.org/" target="_blank"><img src="img/funding/sarcoma.jpg" alt=""/></a>
														</li> 
													</ul>
												</div>
												<div style="clear:both"></div>
											</div>
										</div>
									</div>
								</div>
							</div>
							<?php include "footer.html";?>
							<script type="text/javascript" src="js/jquery.details.js"></script>
							<script type="text/javascript">
							$(function() {
								$('html').addClass($.fn.details.support ? 'details' : 'no-details');
								$('details').details();
							});
							</script>
						</body>
						</html>
